frontend/src/metabase/lib/formatting/numbers.tsx

```diff
@@ -24,7 +24,7 @@ interface FormatNumberOptionsType {
   negativeInParentheses?: boolean;
   number_separators?: string;
   number_style?: string;
-  scale?: string;
+  scale?: number;
```

Comment:
Fix the type, since it's actually a number.

![image](https://user-images.githubusercontent.com/1937582/203581424-345b044b-fc6b-4c60-9a3d-e7883225c348.png)